Helper/Entity/ProductHelper.php

```diff
@@ -41,7 +41,7 @@ class ProductHelper
     private $eventManager;
     private $visibility;
     private $stockHelper;
-    private $stockRegistry;
+    protected $stockRegistry;
```

Comment:
There is a reason why we need to alter the visibility here?

Reply:
To be overriden by the inventory module, as discussed ;)